aiida/transport/plugins/ssh.py

```diff
@@ -483,7 +485,7 @@ def open(self):
         # Open a SSHClient
         connection_arguments = self._connect_args
         proxystring = connection_arguments.pop('proxy_command', None)
```

Comment:
Also, update this to e.g. `proxystring = connection_arguments.pop('proxy_command', '')`

Reply:
But we should really use `None`. When I was fixing this locally I did not manage to do that due to the fact that we have to leave options for the user that has previously defined a string to set it to `None`. Since the cli does not suggest the string in the entry field, one cannot delete it and then set `None` internally. Also, it is not so user friendly to set `None` in the cli.